tests/TestBaseClass.php

```diff
@@ -11,12 +11,41 @@ class TestBaseClass extends TestCase
      */
     protected static $testHelper = null;
 
-    public function __construct()
```

Comment:
The __construct seems to mess up PHPUnits some funtionality. eg the DataProvider.

Reply:
https://stackoverflow.com/questions/10175414/phpunit-dataprovider-simply-doesnt-work

Reply:
Are you calling parent construct?